Libraries/LibJS/Runtime/MathObject.cpp

```diff
@@ -125,10 +125,13 @@ JS_DEFINE_NATIVE_FUNCTION(MathObject::floor)
 JS_DEFINE_NATIVE_FUNCTION(MathObject::ceil)
 {
     auto number = interpreter.argument(0).to_number(interpreter);
+    auto number_double = number.as_double();
     if (interpreter.exception())
         return {};
     if (number.is_nan())
         return js_nan();
+    if (number_double < 0 && number_double > -1)
+        return Value(-0.f);
```

Comment:
```suggestion
        return Value(-0);
```

Reply:
The test doesn't work if I change it to that.

Reply:
huh - `-0.0` then? Either way, there's no float overload of `JS::Value`.